app.ts

```diff
@@ -878,7 +879,7 @@ app.get(
 			scales: scales,
 		}
 
-		let ports: SerialPort.PortInfo[]
+		let ports: PortInfo[]
```

Comment:
You could avoid having to import `@serialport/bindings-cpp` at all if you did this:
```js
try {
  data.serialports = (await SerialPort.list()).map((port) => port.path);
} catch (error) {
  logger.error(error)
  data.serialports = [];
}

res.json(data)
```